com.usebottles.bottles.elementary.json

```diff
@@ -5,31 +5,15 @@
   "sdk": "io.elementary.Sdk",
   "command": "bottles",
   "finish-args": [
-        "--persist=.Bottles",
         "--allow=multiarch",
         "--share=network",
         "--share=ipc",
         "--socket=wayland",
         "--socket=fallback-x11",
         "--socket=pulseaudio",
-        "--filesystem=~/Games:ro",
-        "--filesystem=~/wineprefixes:ro",
-        "--filesystem=~/.Bottles:ro",
-        "--filesystem=~/.local/share/bottles:ro",
```

Comment:
@mirkobrombin the `BottlesFlatpakMigration` does not appear to be used, `show_flatpak_migration_view` is never called

Should I remove it?

Reply:
BottlesFlatpakMigration is no more needed and can be safely removed. It's ok for the section in the readme